documentation/modules/rn-2.6.adoc

```diff
@@ -0,0 +1,405 @@
+
+.Integration of the create and update provider user interface
+
+The user interface of the source providers and destination providers now aligns with the look and feel of the {ocp} web console and displays the current data.
+
+.Standalone UI
+
+The old UI of {project-short} 2.3 cannot run by setting `feature_ui: true` in ForkliftController anymore.
+
+.Improved errors logged in populator pods
+
+In earlier releases of {project-short} {project-version}, populator pods were always restarted on failure. This made gathering logs from the failed pods difficult. {project-short} 2.5.3 has a maximum number of three restarts of populator pods. On the third and final time, the populator pod remains in the fail status, and its logs can then be easily gathered by `must-gather` and by `forklift-controller`, to know this step has failed. link:https://issues.redhat.com/browse/MTV-818[(MTV-818)]
+
+.The `VirtualMachineClusterFlavor` CRDs replace {ocp-name} templates
+
+The Virtual Machine Flavor API replaces {ocp-name} templates. {project-short} currently falls back to using {ocp-name} templates when a relevant preference is not available.
+
+.Full support for migration from OVA
+
+{project-short} {project-version} supports migration from OVA.
+
+.{ocp-name} Console `Create` and `Edit` pages replace Forms and Wizards
+
+Forms and Wizards that use `@migtools/lib-ui` do not function as expected when running inside the plugin. The migtool library was built as a standalone application. The `Create` and `Edit` pages in the {ocp-name} Console replace the Forms and Wizards. link:https://issues.redhat.com/browse/MTV-615[(MTV-615)]
+
+.Scan OVA files in the root folder of an NFS share
+
+Migration scans the files directly to the NFS share folder, not to a separate folder. link:https://issues.redhat.com/browse/MTV-680[(MTV-680)]
+
+
+[id="new-features-and-enhancements-26_{context}"]
+== New features and enhancements
+
+This release has the following features and improvements:
+
+.Replace templates with flavors
+
+KubeVirt API introduces `VirtualMachineClusterFlavor` and `VirtualMachineFlavor` Containerized Data Importers(CDIs). Flavors are meant to replace the Templates, which are only available on {ocp-name}. link:https://issues.redhat.com/browse/MTV-206[(MTV-206,] link:https://issues.redhat.com/browse/MTV-919[MTV-919)]
+
+.Integrate volume populators in remote migrations
+
+// Arik, please check and explain - `el9` > "Red Hat Enterprise Linux 9"?
+Integrate volume populator for `oVirt` and OpenStack, together with `virt-v2v` on `el9` into a CDI to migrate remote {ocp-name} clusters. link:https://issues.redhat.com/browse/MTV-721[(MTV-721)]
+
+.VM posted with `Running` state
+
+{project-short} shows each VM's migration `Running` state during the migration. link:https://issues.redhat.com/browse/MTV-794[(MTV-794)]
+
+.Review and configure migrated VMs
```

Comment:
I didn't realize which change this refers to but looking at MTV-843, it can be documented as a technical change (the migrated VMs are set with a label `app: <vmname>` to ease further changes for them in OpenShift Virtualization)

Reply:
moved to technical changes, lines 49-51